docs/modules/ROOT/pages/highlights.adoc

```diff
@@ -0,0 +1,9 @@
+= What's new in Play SOAP 2.x
+
+This page highlights the new features of Play SOAP 2.x.
+
+== CXF 4.0
+CXF was updated to the new major version 4.0. As a result the `javax.\*` packages were migrated to `jakarta.*` because CXF 4.0 was based on https://jakarta.ee/specifications/platform/9.1/[JakartaEE 9.1]. Fore more details see the https://cxf.apache.org/docs/40-migration-guide.html[CXF 4.0 release notes]
```

Comment:
```suggestion
We have upgraded CXF to the next major version, `4.0`. Because CXF `4.0` is based on https://jakarta.ee/specifications/platform/9.1/[JakartaEE 9.1], all references to the `javax.\*` classpath needed to be migrated to `jakarta.*`. For more details, see the https://cxf.apache.org/docs/40-migration-guide.html[CXF 4.0 release notes]
```